docs/source/formulations_example.rst

```diff
@@ -0,0 +1,648 @@
+        scope=scope
+    )
+    quantity_var = IngredientQuantityInOutput(
+        name=f"{ing_name} mass",
+        headers=[ing_name, "Mass"],
+        process_templates=[mix_template_link],
+        ingredient_name=ing_name,
+        quantity_dimension=IngredientQuantityDimension.MASS,
+    )
+    labels_var = IngredientLabelsSetInOutput(
+        name=f"{ing_name} labels",
+        headers=[ing_name, "Labels"],
+        process_templates=[mix_template_link],
+        ingredient_name=ing_name
+    )
+    config = config.add_columns(
+        variable=identity_var, columns=[IdentityColumn(data_source=identity_var.name)]
+    )
+    config = config.add_columns(
+        variable=quantity_var, columns=[MeanColumn(data_source=quantity_var.name)],
+    )
+    config = config.add_columns(
+        variable=labels_var,
+        columns=[
+            ConcatColumn(
+                data_source=labels_var.name,
+                subcolumn=IdentityColumn(data_source=labels_var.name)
+            )
+        ]
+    )
+
+The figure below shows how these three columns may be rendered in a GEM Table.
+
+.. figure:: _static/GEM_Table_formulation_ingredient.png
+    :align: center
+
+    GEM Table "simple syrup ingredient" columns
+
+Lastly, we register the configuration and build the table.
+Note that this is a long-running process.
+To build the table asynchronously, use :func:`~citrine.resources.gemtables.GemTableCollection.initiate_build`.
+
+.. code-block:: python
+
+    config = project.table_configs.register(config)
+    table = project.tables.build_from_config(config)
+
+Training a Predictor
+--------------------
+
+With the GEM Table in hand, we build and train a predictor to predict the tastiness of novel margarita recipes.
+The first step is to define a :class:`~citrine.informatics.data_sources.GemTableDataSource` based on the Gem Table, ``table``.
+We choose to define a :class:`~citrine.informatics.descriptors.FormulationDescriptor` to hold the formulation;
+if we do not specify it then a default descriptor will be generated, but given how crucial this descriptor is it is best to specify it directly.
+
+.. code-block:: python
+
+    from citrine.informatics.descriptors import FormulationDescriptor
+    from citrine.informatics.data_sources import GemTableDataSource
+
+    formulation = FormulationDescriptor("mixed and blended margarita")
+    data_source = GemTableDataSource(table_id=table.uid, table_version=table.version, formulation_descriptor=formulation)
+
+The first component of the graphical model is a :class:`~citrine.informatics.predictors.simple_mixture_predictor.SimpleMixturePredictor`, which flattens the input formulation--it repeatedly replaces components with their ingredients until only the basic ingredients remain.
+This flattening efficiently teaches the predictor about the relationship between ingredients.
+In this case, it learns exactly how "simple syrup A" and "simple syrup B" are similar to each other.
```

Comment:
MAYBE - "each other by reducing them to their atomic ingredients: water and sugar." 

I'm not sure how ubiquitous the term "flattening" is - it might be helpful to users to explain what's going on a couple of different ways.